Traceback (most recent call last):
  File "bco_editor", line 99, in search_objects
    print doc["provenance_domain"]["contributors"]
KeyError: 'contributors'
